test/test_nestedtensor.py

```diff
@@ -559,9 +559,12 @@ def test_nested_namespace(self):
         nested_namespace_result = torch.nested.to_padded_tensor(nt, 4)
         self.assertEqual(result, nested_namespace_result)
 
-    def test_to(self):
+    @parametrize("layout", [torch.strided, torch.jagged], name_fn=layout_name)
+    def test_to(self, layout):
```

Comment:
nit: the test name doesn't quite match the test logic (e.g. `test_to_jagged()` will actually test the jagged -> strided conversion)

Reply:
I guess the issue is that it's testing all the the .to() semantics with jagged tensors, including the one where it gets converted to strided, and viceversa, will change the name_fn to be more expressive